corehq/apps/export/templates/export/spec/mocha.html

```diff
@@ -1,15 +1,4 @@
 {% extends 'mocha/base.html' %}
 {% load hq_shared_tags %}
 
-{% block dependencies %}
-  <script src="{% static 'export/js/const.js' %}"></script>
-  <script src="{% static 'export/js/utils.js' %}"></script>
-  <script src="{% static 'export/js/models.js' %}"></script>
-{% endblock %}
-
-{% block mocha_tests %}
-  <script src="{% static 'export/spec/data/export_instances.js' %}"></script>
-  <script src="{% static 'export/spec/ExportInstance.spec.js' %}"></script>
-  <script src="{% static 'export/spec/ExportColumn.spec.js' %}"></script>
-  <script src="{% static 'export/spec/Exports.Utils.spec.js' %}"></script>
-{% endblock %}
+{% js_entry_b3 'export/spec/main' %}
```

Comment:
Note this uses B3 even though locations is on B5. All mocha templates are on B3 unless the test name includes `bootstrap5` (e.g., `hqwebapp/bootstrap5`, see [here](https://github.com/dimagi/commcare-hq/blob/98629eaea9bb2cd9984fc374eb25d3dd3a37d87a/Gruntfile.js#L30-L41) for the list of all tests). The test name becomes a URL param [here](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/mocha/urls.py), and the bootstrap version is ultimately set [here](https://github.com/dimagi/commcare-hq/blob/98629eaea9bb2cd9984fc374eb25d3dd3a37d87a/corehq/apps/mocha/views.py#L13-L21).

Reply:
This is why https://github.com/dimagi/commcare-hq/pull/35614/commits/5f1cab262a01abaa7e6a1140850177ec316533d3 is needed.